backend/experiment/forms.py

```diff
@@ -119,6 +120,45 @@
 ]
 
 
+class ModelFormFieldAsJSON(ModelMultipleChoiceField):
+    """ override clean method to prevent pk lookup to save querysets """
+    def clean(self, value):
+        return value
+
+
+class ExperimentSeriesForm(ModelForm):
+    def __init__(self, *args, **kwargs):
+        super(ModelForm, self).__init__(*args, **kwargs)
+        experiments = Experiment.objects.all()
+        self.fields['first_experiments'] = ModelFormFieldAsJSON(
+            queryset=experiments,
+            required=False,
+            help_text=('This field will be deprecated in the nearby future. '
+                       'Please use experiment series groups (see bottom of form).')
+        )
+        self.fields['random_experiments'] = ModelFormFieldAsJSON(
+            queryset=experiments,
+            required=False,
+            help_text=('This field will be deprecated in the nearby future. '
+                       'Please use experiment series groups (see bottom of form).')
+        )
+        self.fields['last_experiments'] = ModelFormFieldAsJSON(
+            queryset=experiments,
+            required=False,
+            help_text=('This field will be deprecated in the nearby future. '
+                       'Please use experiment series groups (see bottom of form).')
+        )
+        self.fields['dashboard'].help_text = ('If you check "dashboard", the experiment collection will have a '
```

Comment:
This field will also be deprecated.

Reply:
Is that because there will always be a dashboard for a collection?

Reply:
The dashboard is going to be on the group level, not the series level. At least I cannot see a use case in which we would want to have a dashboard for the series *and* a group within the series.

Reply:
I see. But the description, about page, logos, and so on will still be part of the experiment series, right?